tests/opentelemetry-docker-tests/tests/celery/test_celery_functional.py

```diff
@@ -558,6 +558,8 @@ class CelerySubClass(CelerySuperClass):
     ) == async_run_span.attributes.get(SpanAttributes.MESSAGING_MESSAGE_ID)
 
 
+# FIXME find a permanent solution for the flakiness of this test
```

Comment:
If the fix for the issue includes a FIXME, then I agree w/ @owais that this doesn't fix #638 😄

Reply:
I have updated the description of this PR :+1: